model/model.js

```diff
@@ -0,0 +1,86 @@
+let fs = require('fs')
+let View = require("../view/view.js")
+
+class Model {
+    constructor(
+        ){
+        this.ID = 0
+        this.nama = 2
+        this.task = 3
+        this.status = 4
+        console.log(this.nama)
+
+    }
+
+    static data(){
+    let fs = require('fs')
+    let Beta = fs.readFileSync('./data/data.json','utf8')
+    let Gamma = JSON.parse(Beta)
+
+    return Gamma
```

Comment:
setelah JSON.parse, data sebaiknya diubah dari array of object menjadi array of instance, baru kemudian di-return